lib/AWS/XRay.pm

```diff
@@ -190,6 +190,30 @@ sub add_capture {
     }
 }
 
+if ($ENV{LAMBDA_TASK_ROOT}) {
+    # AWS::XRay is loaded in AWS Lambda worker.
+    # notify the Lambda Runtime that initialization is complete.
+    mkdir '/tmp/.aws-xray' or warn "failed to make directory: $!";
+    open my $fh, '>', '/tmp/.aws-xray/initialized' or warn "failed to create file: $!";
+    close $fh;
+    utime undef, undef, '/tmp/.aws-xray/initialized' or warn "failed to touch file: $!";
```

Comment:
this code is from https://github.com/aws/aws-xray-sdk-python/blob/dcb08012532d12403df0ec698c7158ab98a45878/aws_xray_sdk/core/lambda_launcher.py#L19-L40